### DISCUSSION TITLE
ask for advice

### DISCUSSION THREAD

**Discussion by magic-core** [krasa/GrepConsole#232] [General]
![image](https://user-images.githubusercontent.com/46321977/173389080-98c9da0f-732c-4f8a-8f9b-d330ac9b4c45.png)

How to achieve this effect?

**Comment by magic-core** [comment]
Idea Plug-in development..........

**Reply by krasa** [reply]
![image](https://user-images.githubusercontent.com/1160875/173390116-d62b92d3-a932-4bd2-8de8-2086a7fcd748.png)
